carbon-route.html

```diff
@@ -223,16 +223,16 @@
     __resetProperties: function() {
       this._setActive(false);
       this._matched = null;
-      //this.tail = { path: null, prefix: null, queryParams: null };
-      //this.data = {};
+      this.tail = { path: null, prefix: null, __queryParams: null };
+      this.data = {};
```

Comment:
Tangent, but it would probably be worth documenting the rationale for not setting `tail` and `data` here in a comment.